### DISCUSSION TITLE
Enabling Disabled Items?

### DISCUSSION THREAD

**Discussion by Reioflight** [NillerMedDild/Enigmatica6#2690] [Q&A]
I am wondering how to enable mainly one disabled item. It is Tha Ghost from the mod Lost Trinkets. Recently I have gotten into game script type stuff, but I am nowhere near proficient enough to screw with anything. Can someone please point me on where to go/what to change to enable Tha Ghost item?

**Comment by MuteTiefling** [comment]
These are disabled directly in the Lost Trinkets config file.
